src/utils/transforms_test.cljs

```diff
@@ -0,0 +1,37 @@
+(ns utils.transforms-test
+  (:require
+    [cljs.test :refer [are deftest is testing]]
+    [utils.transforms :as sut]))
+
+(defn map=
+  [m1 m2]
+  (= (js/JSON.stringify (clj->js m1))
+     (js/JSON.stringify (clj->js m2))))
+
```

Comment:
I know this is just for testing, but the name `map=` is not clear enough because it's like a weak comparison of maps translated to the JS world. Maybe we could add a docstring?

Wdyt about `map-js=` or something similar? 🤔

Since `{:a :b}` will be equal to `{"a" "b"}`.

Reply:
It is flawed. I can give it a more specific name to clarify or docstring. Thanks for the feedback.